plugins/node/opentelemetry-instrumentation-aws-sdk/README.md

```diff
@@ -96,8 +96,8 @@ AWS contains dozens of services accessible with the JS SDK. For many services, t
 
 Specific service logic currently implemented for:
 
-- [SQS](./docs/sqs.md)
-- [SNS](./docs/sns.md)
+- [SQS](./doc/sqs.md)
+- [SNS](./doc/sns.md)
 - DynamoDb
```

Comment:
I just merged the PR that adds lambda instrumentation, seems the link for that is broken too.
Could be great if you can fix that as well when you merge main into this PR

Reply:
rebased and forced pushed to get also a correct commit message